ddmrp/security/ir.model.access.csv

```diff
@@ -15,7 +15,8 @@ access_stock_buffer_procurements_item,stock.buffer.procurements.item.user,model_
 access_stock_buffer_procurements_item_manager,stock.buffer.procurements.item.manager,model_make_procurement_buffer_item,stock.group_stock_manager,1,1,1,1
 access_ddmrp_duplicate_buffer,ddmrp.duplicate.buffer,model_ddmrp_duplicate_buffer,stock.group_stock_user,1,0,0,0
 access_ddmrp_duplicate_buffer_manager,ddmrp.duplicate.buffer,model_ddmrp_duplicate_buffer,ddmrp.group_stock_buffer_maintainer,1,1,1,1
-access_mrp_bom_change_location_manager,mrp.bom.change.location,model_mrp_bom_change_location,stock.group_stock_user,1,1,1,1
```

Comment:
This commit was actually fwported, and this part was changed in this commit just after this one https://github.com/OCA/ddmrp/commit/80ae299de3d83a3e2882f27980f9322a599567ef#diff-370f54b95100e1e7b4cc81f36b5c75e5599f49931925554ac6d5df7e12e2f70aL18

Reply:
Ah yes! Removed the commit